src/mson.js

```diff
@@ -0,0 +1,238 @@
+/*
+ * Renders refract elements into MSON.
+ */
+
+/*
+ * Indent a piece of multiline text by a number of spaces.
+ * Setting `first` to `true` will also indent the first line.
+ */
+function indent(input, spaces, options = {first: false}) {
```

Comment:
Good catch. These were written at different times and do the exact same thing, so I've refactored the code to reuse a single function.